js/cardpopup.js

```diff
@@ -0,0 +1,100 @@
+const tip = tippy( '.cardname', {
+	arrow: false,
+	animateFill: false,
+	followCursor: true,
+	html: '#js--card-popup',
+	placement: 'top',
+	touchHold: true,
+	delay: [ 50, 0 ],
+	animation: 'fade',
+	duration: 0,
+	performance: true,
+	theme: 'scryfall',
+	onShow() {
+		const thisPopper = this,
+			content = thisPopper.querySelector( '.tippy-content' ),
+			/* eslint no-underscore-dangle: ["error", { "allow": ["_reference"] }] */
+			target = thisPopper._reference,
+			jsonURI = new URL( 'https://api.scryfall.com/cards/named' );
+		var rotationClass = 'rotate-0';
+		jsonURI.searchParams.set( 'exact', target.dataset.cardName );
+		jsonURI.searchParams.set( 'set', typeof target.dataset.cardSet === 'undefined' ? '' : target.dataset.cardSet );
+		if ( tip.loading || content.innerHTML !== '' ) { return; }
+		tip.loading = true;
+		// Hide the tooltip until we've finished loaded the image
+		thisPopper.style.display = 'none';
+		// fetch() only works on modern browsers
+		fetch( jsonURI )
+			.then( response => {
+				if ( !response.ok ) { throw Error( response.statusText ); }
+				{ return response; }
+			} )
+			.then( response => response.json() )
+			.then( data => {
+				const queryURI = new URL( target.href ),
+					directURI = new URL( data.scryfall_uri ),
+					utmSource = queryURI.searchParams.get( 'utm_source' );
+				directURI.searchParams.set( 'utm_source', utmSource );
```

Comment:
It's not a written policy. @csuhta requests this on an individual basis.

Reply:
Hey there: I'm a member of the @Scryfall team. Please set `utm_source=whatsinstandard` for any traffic you send our way please. ❤️

Reply:
FYI: In a subsequent commit I just hardcoded the `utm_source`. (We should still set it manually on each homepage link in case clientside javascript is disabled.)